mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -36,12 +36,14 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
 
     # read parameters from config file
     casename = config.get('case', 'casename')
-    ref_casename_v0 = config.get('case', 'ref_casename_v0')
-    indir_v0data = config.get('paths', 'ref_archive_v0_ocndir')
+    ref_casename_v0 = config.get('case', 'ref_casename')
+    indir_v0data = config.get('ocean_reference', 'basedir')
```

Comment:
This should be something like:
`ref_casename_v0 = config.get('case', 'ref_casename_v0')
indir_v0data = config.get('ocean_reference', 'basedir_v0')
ref_casename = config.get('case', 'ref_casename')
indir_refdata = config.get('ocean_reference', 'basedir')`
(with the understanding that at the moment we don't use ref_casename and indir_refdata)